iroh/src/node.rs

```diff
@@ -540,24 +547,31 @@ impl Future for Node {
 }
 
 #[derive(Debug, Clone)]
-struct RpcHandler {
-    inner: Arc<NodeInner>,
+struct RpcHandler<D> {
+    inner: Arc<NodeInner<D>>,
 }
 
-impl RpcHandler {
+impl<D: BaoCollection> RpcHandler<D> {
     fn rt(&self) -> runtime::Handle {
         self.inner.rt.clone()
     }
 
+    fn concrete_db(&self) -> Option<Database> {
```

Comment:
Can you add a doc comment to this?

Reply:
My recollection slowly comes back to me. This is a bit of a hack. Some RPC calls only really make sense if you have not a generic BaoCollection but the concrete `Database`.

So some RPC calls have dummy impls unless you have a `Database`. Yeah, I know, not great. But the alternative would have been coming up with traits for all the things, even things that don't really make sense anymore.